app/assets/stylesheets/application.scss

```diff
@@ -33,6 +33,17 @@ footer {
   padding-top: 15px;
 }
 
+.page-nav {
+  background-color: #808080;
+  color: #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.